fog/sample-paykit/src/cached_tx_data/mod.rs

```diff
@@ -346,10 +366,14 @@ impl CachedTxData {
     /// Poll for new txo data, given fog view connection object
     ///
     /// This is called when doing a balance check
-    pub fn poll_fog_for_txos(&mut self, fog_view_client: &mut FogViewGrpcClient) -> Result<()> {
+    pub fn poll_fog_for_txos(
+        &mut self,
+        fog_view_client: &mut FogViewGrpcClient,
+        fog_block_client: &mut FogBlockGrpcClient,
+    ) -> Result<()> {
```

Comment:
not for this PR but for a later PR maybe:

* this function always returns `Ok(())`
* usually in this function it is dangerous to use `?` operator which is normally how errors should be handled. the reason is that if we early return from this function we might fail to append new tx outs or missed blocks records to our lists, and we would never get them again from fog, so we would eventually get the wrong balance.
* instead it just captures any errors and logs them

it might be better to make this function not return result at all, because then using `?` without thinking about it will not compile